test/api/v01/api_test.rb

```diff
@@ -16,9 +16,11 @@
 # <http://www.gnu.org/licenses/agpl.html>
 #
 require './test/test_helper'
+require './test/api/v01/helpers/count_helper'
 
 class Api::V01::ApiTest < Minitest::Test
   include Rack::Test::Methods
+  include TestHelper
```

Comment:
When do we need this `include TestHelper`? It exists only in 3 test files and I suspect it might not be necessary since we `require './test/test_helper'`?

Reply:
I think this include is not necessary, I had it because there was the same include in `vrp_test.rb`, but it should be removed according to me, I repush

Reply:
this include is mandatory

Reply:
Hmm, it is weird that few test files needed while most of it can do without it. We will look into it when we have time